lean4-infoview/src/infoview/info.tsx

```diff
@@ -245,7 +245,7 @@ function InfoAux(props: InfoProps) {
         setStatus('updating');
 
         let allReq = undefined
-        if (sv.hasWidgetsV1()) {
+        if (sv?.hasWidgetsV1()) {
```

Comment:
Very minor, but I would use `sv!` here keeping in mind the invariant that the message UI is not rendered if `sv === undefined`.